packages/govtnz-ds-website/src/components/layout.tsx

```diff
@@ -22,8 +23,12 @@ const Layout = (props: Props) => {
 
   useEffect(() => {
     if (document.body.classList) {
-      document.documentElement.classList.add('theme-default');
+      document.documentElement.classList.add('theme-default', 'js');
     }
+
+    window.onbeforeprint = () => {
+      generatePrintLinksList();
```

Comment:
From reading MDN I'm unclear on whether Safari supports this.. any ideas?

https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeprint

I wonder if we could generate it regardless and just reveal it in Print CSS?

Reply:
Hmm, well it seems to work for me on Safari:
![page 1](https://user-images.githubusercontent.com/1134713/88997148-1b268400-d343-11ea-8ec6-4c0da3e2fe1c.png)


![page 2](https://user-images.githubusercontent.com/1134713/88997156-1feb3800-d343-11ea-8bdd-16ced2e66a06.png)

forgive tiny preview, here's a closer look:
![Screen Shot 2020-07-31 at 3 33 35 PM](https://user-images.githubusercontent.com/1134713/88997201-3b564300-d343-11ea-80cd-dcf84ced9f80.png)